client/app/components/packages/applicant-team-editor.js

```diff
@@ -20,7 +20,8 @@ export default class ApplicantTeamEditorComponent extends Component {
   @action
   removeApplicant(applicant) {
     // remove the applicant from the ember store
-    this.store.deleteRecord(applicant);
+    // this.store.deleteRecord(applicant);
+    applicant.destroyRecord();
 
     // remove the DOM node by removing the object from the array passed to the editor component
     this.args.applicants.removeObject(applicant);
```

Comment:
This smells not Ember-ish.
```
    this.args.applicants.removeObject(applicant);
```

Reply:
Hmm what do you mean?

Reply:
destroyRecord is a method on ember models

Reply:
also what is "ember-ish"?

Reply:
Oh I was referring to line 27. It modified the arguments to the component, as opposed to Data Down Action Up. That's interesting we have to do that manual cleanup